code/ethan/python/lab07.py

```diff
@@ -0,0 +1,114 @@
+# Compute Automated Readability Index
```

Comment:
[Lab 7: Compute Automated Readability Index](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/07%20ARI.md)

Reply:
I get this error every time I type the file path like that.
    f = open("./we-shall-fight-on-the-beaches.txt", 'r', encoding='utf-8')
FileNotFoundError: [Errno 2] No such file or directory: './we-shall-fight-on-the-beaches.txt'

Reply:
nevermind, figured it out